src/network_manager.rs

```diff
@@ -130,6 +130,8 @@ impl ActiveConnections {
                     connection_type,
                 },
             );
+            self.connection_queue
```

Comment:
Shouldn't be there see review comment on peer.rs

Reply:
done